### DISCUSSION TITLE
Update documentation and examples

### DISCUSSION THREAD

**Discussion by James-Burgess** [jgile/vue-csv-import#80] [General]
Hey @jgile,
I'm looking to do some more work on the lib this [hacktoberfest](https://hacktoberfest.digitalocean.com/)

How do you feel about setting up some docs on [readthedocs](https://readthedocs.org/) or [docisfy.js](https://docsify.js.org/)?
I'll also be making some PR's to show more examples of how to use VCI. (the table mapper, adding styles, adding slots)

Let me know if you'd like me to jump onto anything else :wave:

**Comment by ScottyRobinson** [comment]
Hey @James-Burgess,

Did you get anywhere with this? 

I'd love to see some examples on how you style the input & your own table map.

**Reply by James-Burgess** [reply]
Hey @ScottyRobinson, Thanks for pinging me on this. 

I still have some work in progress and will make an PR soon with some updates the the readme, however, John is usually quite slow to merge.

In the mean time, you can have a look at [This Repl](https://replit.com/@JimmyBurgess/Vue-csv-import-demo) which styles both the input and the table map in a naive multi-step style approach.
